cinema/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.

```

Comment:
Consider implementing test cases for the `Movie` model and any views or serializers you have. This will help ensure your application behaves as expected and can catch potential bugs early.